src/D2662.tex

```diff
@@ -876,7 +890,7 @@ \section{Formal Wording}
 For a template argument \tcode{T}, if the \grammarterm{constant-expression} of a \grammarterm{pack-index-specifier} is type-dependent,
 \tcode{T...[\grammarterm{constant-expression}]} denotes a unique dependent type.
 Two such \grammarterm{pack-index-specifier}{s} refer to the same type only if their \grammarterm{constant-expression}{s} are
-equivalent\iref{temp.over.link} and their type are the same.
+equivalent \iref{temp.over.link} and their type are the same. %lah: what does the "their" refer to? The pack-index-specifiers or the constant-expressions? Also, we would need to alter this to "their types are the same" or "their type is the same".
```

Comment:
the two \grammarterm{pack-index-specifier}{s}

Reply:
I'd add numbering to this to be excruciatingly clear: Two such \grammarterm{pack-index-specifier}{s} refer to the same type only if (1) their \grammarterm{constant-expression}{s} are
equivalent\iref{temp.over.link} and (2) their types are the same.